Documentation/data-collection.md

```diff
@@ -1,8 +1,8 @@
 # OpenShift 4 Data Collection
 
-Red Hat values our customer experience and privacy. It is important to us that our customers understand exactly what we are sending back to engineering and why. During the developer preview or beta release of our software, we want to be able to make changes to our designs and coding practices in real-time based on customer environments. The faster the feedback loop during these development stages the better.
+Red Hat values our customers' experience and privacy. It is important to us that our customers understand exactly what we are sending back to Red Hat Engineering and why. We want to be able to make changes to our designs and coding practices rapidly, based on customer environments. The faster the feedback loop the better.
```

Comment:
based on customer environments
->
based on customer's environments

Reply:
@juzhao thank you for your review and for highlighting this. I have now resolved this point.